src/lucky/server_settings.cr

```diff
@@ -40,7 +40,8 @@ module Lucky::ServerSettings
     if File.exists?(YAML_SETTINGS_PATH)
       File.read YAML_SETTINGS_PATH
     else
-      raise "Expected config file for the watcher at #{YAML_SETTINGS_PATH}"
+      raise "Expected config file for the watcher at #{YAML_SETTINGS_PATH}. \
+      Did you forget to set LUCKY_ENV to production?"
```

Comment:
I like this. I wonder if maybe we can make it look like the other exceptions so it stands out a bit?

```
raise <<-ERROR
Expected config file for the watcher at #{YAML_SETTINGS_PATH}.

Try this...

  ▸ If this is Production, be sure to set LUCKY_ENV=production
  ▸ If this is Development, ensure the #{YAML_SETTINGS_PATH} file exists

ERROR
```

I'm open to wording, but just tossing out the idea.